ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/passthrough/PassThroughHandler.java

```diff
@@ -50,10 +62,14 @@ public void handle(final RoutingContext context, final JsonRpcRequest request) {
   @Override
   public void handle(final RoutingContext context) {
     final HttpServerRequest httpServerRequest = context.request();
+    final String uri =
+        DEFAULT_URI.equals(httpRequestArgs)
```

Comment:
There's a bigger issue here (which isn't evident from _this_ code base) - but users may invoke a "/login" on ethsigner (which is transparently passed through to Besu) - in this instance I'd expect the URI on the forwarded request to be
`<httpRequestArgs + "/" + httpServerRequest.uri()`

It'd be great to avoid performing the ternary operation on every request - i.e. it should be possible to sort this out during construction, such that the path appending "just works" on every request.

Reply:
Hi @rain-on - I'm thinking of adding a flag for the httpRequestArgs + "/" + httpServerRequest.uri() to determine whether or not we append the uri() component.

Challenge I'm having is that the uri() returns / by default, and that messes up the infura configuration. Would still have if/else statement here.

Open to alternative suggestions

Reply:
I added a flag to support the 3 different cases. I couldn't come up with a way to remove the test, but did make it a lot simplier.

I tried adding a handler to re-route the request in Runner; but this cause a lot of problems (we just need to modify the path but re-route does a re-evaluation of route path and a lot of mess)

I thought about adding additional class, but that's overkill for something small

Note current change does support the httpRequestArgs + / + httpServerRequest.uri()